src/commands/generateTestSuite.ts

```diff
@@ -107,6 +116,14 @@ export class GenerateTestSuite implements VscodeCommand {
       return
     }
 
+    let testSuiteType = ''
+    if (!testSuiteType) {
+      testSuiteType =
+        (await vscode.window.showInputBox({
+          prompt: 'Enter type of test suite (Jest, Mocha, Chai, etc): ',
+        })) || ''
+    }
+
```

Comment:
Wondering if we could store this value long term so the user wouldn't have to input this every time they run the command 🤔

Reply:
Good callout. I'm curious if it will store the same value for all projects? If someone is working on one in a different language, then maybe we don't store it? All of that aside, I liked your idea of reading from some sort of tile. Maybe we make a file that defines the language and test suite? Then if the file is missing we ask them for the language and testing framework

Reply:
Yeah that makes sense 👍